sale_order_general_discount_triple/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Order General Discount Triple",
     "summary": "General discount per sale order with triple",
-    "version": "16.0.1.0.0",
+    "version": "16.0.1.1.0",
```

Comment:
Unless you have migration scripts that depends on this, this is not necessary, done by ocabot

Reply:
I didn't know. Its done